### PR TITLE
[Localization] Translate forgotten phrases sentences

### PR DESCRIPTION
**Pull request by Nolway**
## What are the changes?
Translate forgotten sentences on the phases file.

## Why am I doing these changes?
Because I seen few sentences not translated.

## What did change?
Few sentences in the game are now translatable and available on english ans french.

### Screenshots/Videos
image

## How to test the changes?
Go in game, activate the retry setting and loose a battle.

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/de/battle.ts [hunk 1]
```diff
@@ -61,5 +61,10 @@ export const battle: SimpleTranslationEntries = {
   "useMove": "{{pokemonNameWithAffix}} setzt {{moveName}} ein!",
   "drainMessage": "{{pokemonName}} wurde Energie abgesaugt",
   "regainHealth": "KP von {{pokemonName}} wurden wieder aufgefrischt!",
-  "fainted": "{{pokemonNameWithAffix}} wurde besiegt!"
+  "fainted": "{{pokemonNameWithAffix}} wurde besiegt!",
+  "retryBattle": "Would you like to retry from the start of the battle?",
+  "receivedNewModifier": "You received\n{{modifierName}}!",
+  "unlockedSomething": "{{unlockedThing}}\nhas been unlocked.",
+  "congratulations": "Congratulations!",
+  "beatModeFirstTime": "{{speciesName}} beat {{gameMode}} Mode for the first time!\nYou received {{newModifier}}!",
```
Comment (CodeTappert) [inline src/locales/de/battle.ts:69]:
```suggestion
  "fainted": "{{pokemonNameWithAffix}} wurde besiegt!",
  "retryBattle": "Möchtest du vom Beginn des Kampfes neustarten?", 
  "receivedNewModifier": "Du erhältst\n{{modifierName}}!",
  "unlockedSomething": "{{unlockedThing}}\nwurde freigeschaltet.",
  "congratulations": "Glückwunsch!",
  "beatModeFirstTime": "{{speciesName}} hat den {{gameMode}} Modus zum ersten Mal beendet!\nDu erhältst {{newModifier}}!",
```

File: src/locales/ko/battle.ts [hunk 2]
```diff
@@ -62,4 +62,9 @@ export const battle: SimpleTranslationEntries = {
   "drainMessage": "{{pokemonName}}[[로]]부터\n체력을 흡수했다!",
   "regainHealth": "{{pokemonName}}[[는]]\n기력을 회복했다!",
   "fainted": "{{pokemonNameWithAffix}}[[는]] 쓰러졌다!",
+  "retryBattle": "Would you like to retry from the start of the battle?",
+  "receivedNewModifier": "You received\n{{modifierName}}!",
+  "unlockedSomething": "{{unlockedThing}}\nhas been unlocked.",
+  "congratulations": "Congratulations!",
+  "beatModeFirstTime": "{{speciesName}} beat {{gameMode}} Mode for the first time!\nYou received {{newModifier}}!",
```
Comment (returntoice) [inline src/locales/ko/battle.ts:69]:
```suggestion
  "retryBattle": "이 배틀의 처음부터 재도전하시겠습니까?",
  "receivedNewModifier": "{{modifierName}}[[를]]\n손에 넣었다!",
  "unlockedSomething": "{{unlockedThing}}[[가]]\n해금되었다.",
  "congratulations": "축하합니다!",
  "beatModeFirstTime": "{{speciesName}}[[가]] {{gameMode}} 모드를 처음으로 클리어했다!\n{{newModifier}}[[를]] 손에 넣었다!",
```
Translations for Korean locales

**Review by returntoice — CHANGES_REQUESTED** [review]

File: src/locales/fr/battle.ts
[the remark this answers is not in the document]
Comment (Nolway) [reply src/locales/fr/battle.ts:1]: The modifications is still present but I collapse many commits during a rebase ><

[the remark this answers is not in the document]
Comment (Nolway) [reply src/locales/fr/battle.ts:1]: I fix that

[the remark this answers is not in the document]
Comment (Nolway) [reply src/locales/fr/battle.ts:1]: fixed

[the remark this answers is not in the document]
Comment (Dakurei) [reply src/locales/fr/battle.ts:1]: > fixed

Always in english after your force push
image

File: src/locales/fr/battle.ts [hunk 3]
```diff
@@ -70,4 +70,9 @@ export const battle: SimpleTranslationEntries = {
   "statHarshlyFell": "harshly fell",
   "statSeverelyFell": "severely fell",
   "statWontGoAnyLower": "won't go any lower",
+  "retryBattle": "Would you like to retry from the start of the battle?",
+  "receivedNewModifier": "You received\n{{modifierName}}!",
+  "unlockedSomething": "{{unlockedThing}}\nhas been unlocked.",
+  "congratulations": "Congratulations!",
+  "beatModeFirstTime": "{{speciesName}} beat {{gameMode}} Mode for the first time!\nYou received {{newModifier}}!",
```
Comment (Dakurei) [inline src/locales/fr/battle.ts:77]: Still in English

Comment (Nolway) [reply src/locales/fr/battle.ts]: Now it's ok, sorry about that

File: src/locales/fr/battle.ts [hunk 4]
```diff
@@ -70,4 +70,9 @@ export const battle: SimpleTranslationEntries = {
   "statHarshlyFell": "harshly fell",
   "statSeverelyFell": "severely fell",
   "statWontGoAnyLower": "won't go any lower",
+  "retryBattle": "Voulez-vous réessayer depuis le début du combat ?",
+  "receivedNewModifier": "Vous avez reçu\n{{modifierName}}!",
+  "unlockedSomething": "{{unlockedThing}}\na été débloqué.",
+  "congratulations": "Félicitations!",
+  "beatModeFirstTime": "{{speciesName}} a battu le mode {{gameMode}} pour la première fois!\nVous avez reçu {{newModifier}}!",
```
Comment (Dakurei) [inline src/locales/fr/battle.ts:77]:
```suggestion
  "retryBattle": "Voulez-vous réessayer depuis le début du combat ?",
  "receivedNewModifier": "Vous avez reçu\n{{modifierName}} !",
  "unlockedSomething": "{{unlockedThing}}\na été débloqué.",
  "congratulations": "Félicitations !",
  "beatModeFirstTime": "{{speciesName}} a battu le mode {{gameMode}} pour la première fois !\nVous avez reçu {{newModifier}} !",
```

**Comment by EnochG1** [comment]
LGTM!
I tested in Korean and it works well.
image

**Review by EnochG1 — APPROVED** [review]

**Comment by Nolway** [comment]
> LGTM! I tested in Korean and it works well. image

Nice =)

**Comment by EnochG1** [comment]
@Nolway 
Could you merge branch 'main' into your branch?
I want to continue this PR.
Or if it's difficult for you to continue this PR on your own, I would mirror yours and continue.

**Comment by CodeTappert** [comment]
Please fix the conflicts here!

**Comment by EnochG1** [comment]
Mirror PR created at https://github.com/pagefaultgames/pokerogue/pull/2906

**Comment by Nolway** [comment]
> Mirror PR created at #2906

You create a pr mirroire in 3 days without any answer while I've been waiting for 3 weeks ...

**Comment by EnochG1** [comment]
@Nolway,
I apologize for being radical without thinking deeply.
I thought long and hard before commenting, and I didn't want to see your PR leave it because it was such good work. (I wasn't the only one who think it.)
I don't have any authority here, I'm just one person who wants people like me who are not comfortable with English to enjoy translated games.
My English may not be good enough, but I really hope these PR merged into the main.
And hope we can collaborate in the future if you are okay.

**Pull request closed without merging**